dexios/src/subcommands/header_key.rs

```diff
@@ -142,7 +141,7 @@ pub fn change_key(input: &str, key_old: &Key, key_new: &Key) -> Result<()> {
         HeaderVersion::V5 => {
             let mut keyslots = header.keyslots.clone().unwrap();
 
-            if keyslots.len() < 1 {
+            if keyslots.is_empty() {
```

Comment:
I always forget where `is_empty()` works or not, here is one of them. Nice catch!

Reply:
this is just faster)